std/algorithm/sorting.d

```diff
@@ -19,6 +19,8 @@ $(T2 isPartitioned,
         afterwards.)
 $(T2 isSorted,
         $(D isSorted([1, 1, 2, 3])) returns $(D true).)
+$(T2 isSortedStrictly,
```

Comment:
I initially proposed `isStrictlySorted` - @JackStouffer [recommended](https://github.com/D-Programming-Language/phobos/pull/4025#discussion_r55435550) to change it.